pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -193,10 +193,12 @@ func (r *ReconcileWindowsMachineConfig) reconcileWindowsNodes(desired, current i
 		return true
 	}
 	r.tracker.WindowsVMs(r.windowsVMs)
+	log.V(1).Info("starting tracker reconciliation")
 	err := r.tracker.Reconcile()
 	if err != nil {
 		log.Error(err, "tracking Windows VMs via configmap failed")
```

Comment:
are we changing all the configmap occurences to ConfigMap or just in validation_test.go?

Reply:
We should change this everywhere. I can fix that here too.

Reply:
thank you, I see it one more place in tracker.go file. I could have missed some other occurance.

Reply:
I did a `s/configmap/ConfigMap` across the project and updated the PR and commit messages to reflect that.